src/main/java/com/airlines/dao/CrewDao.java

```diff
@@ -0,0 +1,12 @@
+package com.airlines.dao;
+
+import com.airlines.model.CrewMember;
+
+import java.util.List;
+
+public interface CrewDao {
+
+    List<CrewMember> getCrewMembersByCrewID(int crewID);
```

Comment:
this is holywar topic crewID vs crewId, details for example could be read there:
[https://stackoverflow.com/questions/779929/id-id-or-id](url)

[https://stackoverflow.com/questions/1699944/what-is-correct-java-naming-convention-for-id/1699965](url)

The main point - it should be consistent through project.
If you write your own project: do as you wish.
If there will be reviewers, than discussion could take place
If you join exist project, you just look at what approach ALREADY used in project and just keep same

So on first place, Id/id or ID, but it should be same approach through whole project

As my opinion, approach crewId, id, I saw more often, and almost didn't meet with ID approach in real code, but it's my experience,
so I suggest using crewId in  this project, while keeping in mind reasons behind this, that you can read on provided links